install/README.md

```diff
@@ -1,30 +1,30 @@
-# Install CSM
+# Cray System Management Install Documentation
 
-## Abstract
+This page will guide an administrator through installing Cray System Management (CSM) on an
+HPE Cray EX system.
 
-Installation of the CSM product stream has many steps in multiple procedures which should be done in a
-specific order. Information about the HPE Cray EX system and the site is used to prepare the configuration
-payload. The initial node used to bootstrap the installation process is called the PIT node because the
-Pre-Install Toolkit (PIT) is installed there.
+The CSM services provide essential software infrastructure including the API gateway
+and many micro-services with REST APIs for managing the system.
 
-Once the management network switches have been configured, the other
-management nodes can be deployed with an operating system and the software to create a Kubernetes cluster
-utilizing Ceph storage. The CSM services provide essential software infrastructure including the API gateway
-and many micro-services with REST APIs for managing the system. Once administrative access has been configured,
-the installation of CSM software can be validated with health checks before doing operational tasks
-like the checking and updating of firmware on system components or the preparation of compute nodes.
+Fresh-installations on bare-metal or re-installations of CSM must follow this guide in procedural
+order.
 
-Once the CSM installation has completed, other product streams for the HPE Cray EX system can be installed.
+After completing an installation, the CSM product's installed state will need to be validated
+with various health checks before operational tasks or other product installs (such as
+slingshot) can commence.
 
 ## Topics
 
-1. [Preparing for a Re-installation](#preparing-for-a-re-installation)
+The topics in this chapter need to be done as part of an ordered procedure so are
+shown here with numbered topics.
+
 1. [Pre-Installation](#pre-installation)
-   1. [Boot Installation Environment](#1-boot-installation-environment)
-   1. [Import CSM Tarball](#2-import-csm-tarball)
-   1. [Create System Configuration](#3-create-system-configuration)
-   1. [Configure Management Network Switches](#4-configure-management-network-switches)
-   1. [Identify Inventory](#5-identify-inventory)
+    1. [Preparing for a Re-installation](#1-preparing-for-a-re-installation)
+    1. [Boot Installation Environment](#2-boot-installation-environment)
```

Comment:
I think when we include the number in the header line (like we do for some of these), we should use the actual numbers here instead of the generic ones. That is a reminder to make sure to update the headers if they are changed.